test/spec/controllers/channels.js

```diff
@@ -138,4 +138,47 @@ describe('Controller: ChannelsCtrl', function () {
     modalSpy.should.be.calledOnce;
     httpBackend.flush();
   });
+
+
+  it('should getLowestPriority level from channels', function () {
+    createController();
+    httpBackend.flush();
+
+    var lowestLevel = scope.getLowestPriority();
+    lowestLevel.should.equal(7);
+  });
+
+
+  it('should successfully updateChannelPriority level', function () {
+    createController();
+    httpBackend.flush();
+
+    scope.channels[0].$update = sinon.spy();
+    scope.channels[1].$update = sinon.spy();
+    scope.channels[2].$update = sinon.spy();
+
+    scope.channels[0].should.have.property('priority', 1);
+    scope.channels[1].should.have.property('priority', 5);
+    scope.channels[2].should.have.property('priority', 7);
+
+    // up the priority level - Should stay the same as 1 is the highest
+    scope.updateChannelPriority(scope.channels[0], 'up');
+    scope.channels[0].should.have.property('priority', 1);
+    scope.updateChannelPriority(scope.channels[0], 'down');
+    scope.channels[0].should.have.property('priority', 2);
+
+    scope.updateChannelPriority(scope.channels[1], 'up');
+    scope.updateChannelPriority(scope.channels[1], 'up');
+    scope.channels[1].should.have.property('priority', 3);
+
+    scope.updateChannelPriority(scope.channels[2], 'down');
+    scope.updateChannelPriority(scope.channels[2], 'down');
+    scope.updateChannelPriority(scope.channels[2], 'down');
+    scope.channels[2].should.have.property('priority', 10);   
+
+    scope.channels[0].$update.should.be.called;
+    scope.channels[1].$update.should.be.called;
+    scope.channels[2].$update.should.be.called;
+  });
```

Comment:
Side note: usually unit tests should test only one thing at a time, so this should be broken into multiple tests for each thing it's testing. You could abstract out the common setup stuff into a function and use that in the other tests to make it easier to repeat the tests.

I don't think you need to change this here, it works well as it is as they are pretty straight forward. Just an FYI. :)